frontend/README.md

```diff
@@ -1,6 +1,42 @@
 # Frontend
+UI service for a chat bot.
 
-UI service for InferGPT
+# Run Locally
 
-### To do
-- [ ] Implement initial front end
+Checkout the code, and run the following commands from the root of the frontend directory:
+
+```bash
+npm install
+npm start
+```
+
+The app should now be running on [localhost:3000](http://localhost:8650/).
+
+# Connect to Backend
+
+To connect to the backend you will need to have the correct `.env` file locally. To do this:
+
+- Make a copy of the `.env.example` file in the root of the frontend directory and name it `.env`
+- Update the values in your `.env` file
+
+# Code linting
+
+We have set up ESLint. To run the linter, run the following command:
+
+```bash
+npm run lint
+```
+
+This will attempt to fix any errors you have and will also show you any errors that it can't fix. To automatically fix errors on save (for vscode) do the following:
+
+- create a `.vscode` folder in the root of the project
+- create a `settings.json` file in the `.vscode` folder
+- add the following to the `settings.json` file:
+
+```json
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
+  }
+}
+```
```

Comment:
Ran `npm run lint` and got this error?

![image](https://github.com/WaitThatShouldntWork/InferGPT/assets/48159708/5d9b44b5-c0ae-4caa-b36e-52b9609c2f47)

[According to this](https://stackoverflow.com/questions/72780296/warning-react-version-not-specified-in-eslint-plugin-react-settings-while-run) it needs this config in `eslintrc.json`?

```
{
  "settings": {
    "react": {
      "version": "detect"
    }
  }
}
```

Reply:
yup that got rid of the warning, nice one, thanks